test/distribution/test_mx_distribution_inference.py

```diff
@@ -150,6 +153,8 @@ def maximum_likelihood_estimate_sgd(
                 if not hybridize:
                     assert loss.shape == distr.batch_shape
             loss.backward()
+            # import pdb
+            # pdb.set_trace()
```

Comment:
This change was probably not intended? I think after removing this we can merge.

Reply:
Removed.